src/mongo/integration/standalone/dbclient_test.cpp

```diff
@@ -1414,7 +1414,11 @@ namespace {
         if (serverGTE(&c, 2, 4) && (!serverGTE(&c, 2, 7) || kCompiledWithSSL)) {
             createUser(c, TEST_DB, "user3", "password3");
             std::string errmsg;
-            ASSERT_FALSE(c.auth("test", "user3", "notPassword3", errmsg));
+            try {
+                ASSERT_FALSE(c.auth("test", "user3", "notPassword3", errmsg));
+            } catch (const DBException&) {
+                //Expected on 2.2
```

Comment:
Yeah, I forgot that this was an example. I'd prefer that the examples be standalone so that people can copy and paste them into things and have them run.

I'm as in favor of DRY as possible, but, this duplication is far from the worst thing in this codebase.